go-controller/pkg/cni/ovs_unit_test.go

```diff
@@ -0,0 +1,411 @@
+package cni
+
+import (
+	"fmt"
+	"testing"
+
+	mock_k8s_io_utils_exec "github.com/ovn-org/ovn-kubernetes/go-controller/pkg/testing/mocks/k8s.io/utils/exec"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	kexec "k8s.io/utils/exec"
+)
+
+type onCallReturnArgs struct {
```

Comment:
you may want to use the struct definition at https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/pkg/testing/testifymockhelper.go#L3